packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -259,9 +259,11 @@ export const renderHtml = async (
           {},
           {
             get(_target, name: string) {
-              const file = filePath.slice(config.basePath.length);
+              console.log('filePath', filePath);
```

Comment:
Let's log `wakuDist` in L270 and see if L274 check works the same between ubuntu and windows.

Reply:
sure, i'll try that soon.